packages/mui-joy/src/Button/Button.tsx

```diff
@@ -21,6 +30,8 @@ const useUtilityClasses = (ownerState: ButtonProps & { focusVisible: boolean })
       variant && `variant${capitalize(variant)}`,
       color && `color${capitalize(color)}`,
       size && `size${capitalize(size)}`,
+      elevation && `elevation${capitalize(elevation)}`,
+      roundness && `roundness${capitalize(roundness)}`,
```

Comment:
Another alternative for the prop name - `circularity`. I think it's easier to read and write.

Reply:
🤔 I think we can vote on this before the first release (I still prefer` roundness` because of fewer characters), let's try this out first.

Reply:
Just `radius` is also an option and it could be even easier as it pairs with `border-radius`. 
(👀 [Mantine's going for radius as well](https://mantine.dev/core/button/) 👀)

Reply:
> Just radius is also an option and it could be even easier as it pairs with border-radius.

Hmm, sounds good.

Reply:
+1 for `radius` seems like the simplest :)